### PR TITLE
Search page related #16

### PR DESCRIPTION
**Pull request by RamyAlshurafa**

### CODE REVIEW

File: client/src/Components/CommonComponents/Select/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import ReactSelect from 'react-select';
+
+class Select extends Component {
+  state = {
+    selectedOption: '',
+  }
+  handleChange = (selectedOption) => {
+    this.setState({ selectedOption });
```
Comment (sultanassi95) [inline client/src/Components/CommonComponents/Select/index.js]: use this instead, for consistency and purity manners.
```javascript
    this.setState({
      ...this.state,
      selectedOption
    });
```
read more [here](https://stackoverflow.com/questions/31048953/what-do-these-three-dots-in-react-do)

File: client/src/Components/Pages/ResultPage/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            <h4 className='contract-result-summary__number'>15 Contracts found</h4>
+          </div>
+          <Contract />
+        </div>
+      </main>
+
+    );
+  }
+}
+
```
Comment (sultanassi95) [inline client/src/Components/Pages/ResultPage/index.js]: Could this be a classless component?

**Comment by sultanassi95** [comment]
Good job regarding clean code stuff :clap:

**Comment by AkramBadah** [comment]
I have nothing to add on Soltan. Good job guys regarding the clean code.

**Pull request merged**